.github/actions/setup-python/action.yml

```diff
@@ -23,16 +23,14 @@ runs:
         if [ -z "$python_version" ]; then
           python_version=$(python setup.py -q min_python_version)
         fi
-        if [[ "$python_version" == "3.7" ]]; then
+        if [[ "$python_version" == "3.8" ]]; then
           if [ ${{ runner.os }} == "Windows" ]; then
-            python_version="3.7.9"
+            # GitHub Actions does not have a Python 3.8.12 binary for Windows as of Oct. 27 2022
+            python_version="3.8.10"
           else
-            python_version="3.7.12"
-          fi
-        elif [[ "$python_version" == "3.8" ]]; then
-          python_version="3.8.12"
+            python_version="3.8.12"
```

Comment:
(add in `fi` to close the if/else/fi loop)

Reply:
Thanks @BenWilson2 !